### PR TITLE
withoutRetrofit

### PR DESCRIPTION
**Pull request by ghilbing**
This version without retrofit is working, it is working with activities, I did the adaptation to fragment

### CODE REVIEW

File: app/src/main/java/com/fantastic/bookxchange/fragments/AddBookSearchingFragment.java [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+import java.util.ArrayList;
+
+import cz.msebera.android.httpclient.Header;
+import retrofit2.http.PUT;
+
+
+public class AddBookSearchingFragment extends Fragment {
+
+    private ListView lvBooks;
```
Comment (m3libea) [inline app/src/main/java/com/fantastic/bookxchange/fragments/AddBookSearchingFragment.java]: Better recycler view for this! Actually, I think you can probably reuse the base fragment I created for the books :)

File: app/src/main/java/com/fantastic/bookxchange/fragments/AddBookSearchingFragment.java [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+        View view = inflater.inflate(R.layout.fragment_add_book_searching, container, false);
+
+        lvBooks = view.findViewById(R.id.lvBooks);
+        mBooks = new ArrayList<>();
+        bookTestAdapter = new BookTestAdapter(getContext(), mBooks);
+
+        setupFooter();
+
+        lvBooks.setOnItemClickListener(new AdapterView.OnItemClickListener() {
```
Comment (m3libea) [inline app/src/main/java/com/fantastic/bookxchange/fragments/AddBookSearchingFragment.java]: Probably is better to have some kind of button on the right side of the row. That way the user can also see the detail view of the book clicking on the row.

File: app/src/main/java/com/fantastic/bookxchange/fragments/AddBookSearchingFragment.java (hunk 1 above)
Comment (ghilbing) [reply app/src/main/java/com/fantastic/bookxchange/fragments/AddBookSearchingFragment.java]: Yes, I'll fix it

File: app/src/main/java/com/fantastic/bookxchange/fragments/AddBookSearchingFragment.java (hunk 2 above)
Comment (ghilbing) [reply app/src/main/java/com/fantastic/bookxchange/fragments/AddBookSearchingFragment.java]: ok, great, I'll do it too

Comment (ghilbing) [reply app/src/main/java/com/fantastic/bookxchange/fragments/AddBookSearchingFragment.java]: Thanks for your feedback

**Comment by m3libea** [comment]
Hi @ghilbing , We need to add this too right? can you resolve the conflicts? thank you! :)

**Comment by ghilbing** [comment]
Another PR replace this one

**Pull request closed without merging**